fvp.mk

```diff
@@ -18,23 +18,35 @@ include common.mk
 ################################################################################
 ARM_TF_PATH		?= $(ROOT)/arm-trusted-firmware
 EDK2_PATH		?= $(ROOT)/edk2
-EDK2_BIN		?= $(EDK2_PATH)/Build/ArmVExpress-FVP-AArch64/RELEASE_GCC49/FV/FVP_AARCH64_EFI.fd
+EDK2_BIN		?= $(ROOT)/Build/ArmVExpress-FVP-AArch64/RELEASE_GCC49/FV/FVP_AARCH64_EFI.fd
+EDK2_PLATFORMS_PATH	?= $(ROOT)/edk2-platforms
 FOUNDATION_PATH		?= $(ROOT)/Foundation_Platformpkg
 ifeq ($(wildcard $(FOUNDATION_PATH)),)
 $(error $(FOUNDATION_PATH) does not exist)
 endif
+GRUB_PATH		?= $(ROOT)/grub
+GRUB_CONFIG_PATH	?= $(BUILD_PATH)/fvp/grub
+OUT_PATH		?= $(ROOT)/out
+GRUB_BIN		?= $(OUT_PATH)/bootaa64.efi
+BOOT_IMG		?= $(OUT_PATH)/boot-fat.uefi.img
 
 ################################################################################
 # Targets
 ################################################################################
-all: arm-tf edk2 linux optee-os optee-client xtest optee-examples
-clean: arm-tf-clean busybox-clean edk2-clean optee-os-clean \
+all: arm-tf boot-img edk2 grub linux optee-os optee-client xtest optee-examples
```

Comment:
`boot-img` depends on `linux grub update_rootfs` so can remove `linux` and `grub` from `all:`.

Reply:
That is probably true and actually yesterday I remove every single prereq/target and started from scratch, because I think we have dependencies that are not working as they should. But didn't submit that patch yet, I wanted to wait with that and do that as a separate "clean pre-req / target" thing. We have some platforms where the -jX flag doesn't work on the first run (RPI3).